tests/common.go

```diff
@@ -2060,15 +2389,18 @@ func SetSourceKubeConfig() error {
 	if err != nil {
 		return err
 	}
-	SetClusterContext(sourceClusterConfigPath)
-	return nil
+	err = SetClusterContext(sourceClusterConfigPath)
+	return err
 }
 
 // SetDestinationKubeConfig sets current context to the kubeconfig passed as destination to the torpedo test
-func SetDestinationKubeConfig() {
+func SetDestinationKubeConfig() error {
 	destClusterConfigPath, err := GetDestinationClusterConfigPath()
-	expect(err).NotTo(haveOccurred())
-	SetClusterContext(destClusterConfigPath)
+	if err != nil {
+		return err
+	}
+	err = SetClusterContext(destClusterConfigPath)
+	return err
```

Comment:
To ensure consistency, you could return err as follows:

```
	if err != nil {
		return err
	}
	return nil
```

Reply:
the pattern you suggest (`if err != nil`) is used when you want to early return. But in this case, it makes no sense to early return as **nothing is being done** after the final error is returned, so we can directly return the error.
Your suggestion, in my opinion, would be useful _if_ there was some future scope of modification of function, which seems unlikely. So, I feel we don't need to be consistent in this case as it simply increases code length (and execution time, if the compiler doesn't optimize)

Though, you could tag mithun if you want to confirm if your idea is better or not.